server/src/metadata.rs

```diff
@@ -291,8 +305,20 @@ pub mod entities_metadata_query {
                     })
                     .unwrap_or(Vec::new());
                 let current_date = Utc::now().to_rfc3339();
+                let subject_ids: Vec<i32> = result.subject_ids.as_ref()
+                    .and_then(|value| value.as_array())
+                    .map(|ids| {
+                        ids.iter()
+                            .filter_map(|element| element.as_i64())
+                            .collect::<HashSet<i64>>()
+                            .into_iter()
+                            .map(|id| id as i32)
+                            .collect()
+                    })
+                    .unwrap_or(Vec::new());
 
                 EntityMetadata {
+                    about: subject_ids,
```

Comment:
See https://dini-ag-kim.github.io/amb/draft/#about for the definition. The values must be from https://skohub.io/dini-ag-kim/schulfaecher/heads/main/w3id.org/kim/schulfaecher/index.de.html

Reply:
Should the subjects of other instances also receive the German name?

Reply:
And how shall I procede with subjects which do not have an equivalent subject in the schema, e.g. "Nachhaltigkeit" could be "Nature Conservation" but it's not the same or "Community" which doesn't have any similar subject?

Reply:
> Should the subjects of other instances also receive the German name?

I don't understand you. The values are from a vocabulary and language agnostic. See https://dini-ag-kim.github.io/amb/draft/schemas/about.json for the formal definition and https://github.com/dini-ag-kim/amb/blob/main/draft/examples/valid/highered-course.json#L22 for how they should be added (ignore `prefLabel` here).

Reply:
> And how shall I procede with subjects which do not have an equivalent subject in the schema, e.g. "Nachhaltigkeit" could be "Nature Conservation" but it's not the same or "Community" which doesn't have any similar subject?

I can give you an answer but since you are the product owner of this feature: What do you suggest?

Reply:
My suggestion: If it is close enough to a course, we can point to it, otherwise nullify as @hugotiburtino suggested.

Reply:
> My suggestion: If it is close enough to a course, we can point to it, otherwise nullify as @hugotiburtino suggested.

:+1: (here `nullify` means omit it in the result, right? Then yes!)